assets/benchmarks/shaders/Benchmark_Texture.hlsl

```diff
@@ -20,13 +20,16 @@ RWStructuredBuffer<float> dataBuffer : register(u11);
 float4 psmain(VSOutputPos input) : SV_TARGET
```

Comment:
what is the wave occupancy of this shader? Could you add some comments that indicate the occupancy in this shader? (same for above shaders just to be consistent, thanks :) )